{
  "module": "Test",
  "kind": "Generic Error",
  "text": [
    { "type": "text", "content": "The type " },
    { "type": "code", "content": "Foo" },
    { "type": "text", "content": " expects " },
    { "type": "code", "content": "1" },
    { "type": "text", "content": " type arguments" },
    { "type": "text", "content": ", but I only found " },
    { "type": "code", "content": "2" },
    { "type": "text", "content": " arguments." }
  ],
  "span": {
    "filename": "test-programs/suites/011-bugs/github-359/Test.aum",
    "startp": { "line": 4, "column": 8 },
    "end": { "line": 8, "column": 8 }
  },
  "context": [
    [ 2, "    record Foo[A: Free]: Free is" ],
    [ 3, "        a: A;" ],
    [ 4, "    end;" ],
    [ 5, "" ],
    [ 6, "    function foo(f: Foo[Int32, Int32]): Unit is" ],
    [ 7, "        return nil;" ],
    [ 8, "    end;" ],
    [ 9, "" ],
    [ 10, "    function main(): ExitCode is" ]
  ]
}